src/components/Dropdowns/StyledSelect.tsx

```diff
@@ -15,7 +15,9 @@ type AntStyledSelectProps = StateManagerProps & {
 
 // Styling is done via both styled-components and react-select's `styles` prop.
 const SelectContainer = styled.div<{ $type: ButtonProps["type"] | "outlined" }>`
-  transition: all 0.2s cubic-bezier(0.645, 0.045, 0.355, 1), width;
+  & {
+    transition: all 0.2s cubic-bezier(0.645, 0.045, 0.355, 1), width 0s linear;
+  }
```

Comment:
just curious: what does this `&` do? best I can work out is this will repeat styled-components's generated class name twice - `.abcdef .abcdef {}`?

Reply:
This comment made me dig a little deeper into CSS and realize that this was being overridden! I've updated the selector.
But yes it won't do much besides the selector and I think I added it as a force of habit.